File: java/StriverA-z/basics/Sorting/BubbleSort.java
public class BubbleSort {
    static void Sorter(int [] arr){
        int n=arr.length;
       for(int i=n-1;i>=0;i--){
        for(int j=0;j<i-1;j++){
            if(arr[j+1]<arr[j]){
               int temp=arr[j];
               arr[j]=arr[j+1];
               arr[j+1]=temp;
                
                

            }

        }
       }

    }
    public static void main(String[] args) {
        int [] arr={6,5,2,3,5,8,9};
        Sorter(arr);
        for(int i=0;i<arr.length;i++){
            System.out.println(arr[i]);
        }
      
        
    }
}

File: java/04.sortingAlgos/bubblesort.py
def BubbleSort(arr):
    n=len(arr)
    print(n)
    for i in range(n-1):
       for j in range(n-1):
            if arr[j]>arr[j+1]:
                arr[j], arr[j+1] = arr[j+1], arr[j]

    return arr               

arr=[1,23,4,56,78,234,56,7,8,98]    
print(BubbleSort(arr))